.devfile

```diff
@@ -0,0 +1,24 @@
+specVersion: 0.0.1
+name: che-in-che
+projects:
+  - name: che
+    source:
+      type: git
+      location: 'https://github.com/eclipse/che.git'
+tools:
+  - name: theia-editor
+    type: cheEditor
+    id: org.eclipse.che.editor.theia:1.0.0
+  - name: exec-plugin
+    type: chePlugin
+    id: che-machine-exec-plugin:0.0.1
+  - name: che-dev
+    type: kubernetes
+    local: .che-dev.yaml
+commands:
+  - name: build
+    actions:
+      - type: exec
+        tool: che-dev
+        command: cd /projects/che && mvn clean install -Pnative -DskipTests=true
```

Comment:
I think it makes sense to have two commands - for building with and without tests. Anyway, command name says nothing about skipping tests.